packages/maskbook/src/plugins/Election2020/UI/ElectionPacket.tsx

```diff
@@ -157,6 +158,10 @@ export function ElectionPacket(props: ElectionPacketProps) {
         retry: revalidateTokensOfOwner,
     } = useElectionTokensOfOwner(payload.state, electionToken)
 
+    // fetch images
+    const { value: FireworksImage } = useBase64(getUrl('/Election2020/fireworks.jpg'))
```

Comment:
Please put image in this plugin, then use `new URL("./fireworks.jpg", import.meta.url)` to refer to that image. Don't put more assets in the `/public` folder

Reply:
Not working for me. What should the URL of assets in the plugin folder look like?

Reply:
> ```
> console.log('PoolBackground url', new URL('../assets/pool-background.jpg', import.meta.url).toString())
> 
> // print 
> 
> chrome-extension://jkoeaghipilijlahjplgbfiocjhldnap/f9d2ac0747f4e264148d.jpg
> ```

oh that means if we want to use that directly we need to add it to the "web_accessible_resource" key in our manifet